FlightReservation/App.js

```diff
@@ -24,49 +24,14 @@ import {
   ReloadInstructions,
 } from 'react-native/Libraries/NewAppScreen';
 
+import SignupForm from './components/signupView';
```

Comment:
I think the file would be called as equal as the component that it's exporting for avoiding confusion